### PR TITLE
Update Pulsar PMC members and committers to website

### PR DESCRIPTION
**Pull request by lhotari**
Synchronize team.js with https://whimsy.apache.org/roster/committee/pulsar.json . Downloading requires authentication with Apache ID and that's why it doesn't make sense to automate it further atm. 

This is how I generated team.js:

1. stored https://whimsy.apache.org/roster/committee/pulsar.json to ~/Downloads

2. ran this command to convert pulsar.json to team.js:

```bash
{ echo -n "module.exports = " && cat ~/Downloads/pulsar.json | jq '{"pmc": [.roster| to_entries | sort_by(.key) | .[] | select(.value.role|startswith("PMC")) | {"name":.value.name, "apacheId": .key}], "committers": [.roster| to_entries | sort_by(.key) | .[] | select(.value.role=="Committer") | {"name":.value.name, "apacheId": .key}]}' } | perl -pe 's/$/;\n/ if eof' > team.js
```

In addition, I made changes to show both PMC member and committers on https://pulsar.apache.org/community/#section-community and https://pulsar.apache.org/team/ pages.

### CODE REVIEW

**Comment by lhotari** [comment]
Preview of community page changes: 
image

File: site2/website-next/src/pages/community.js [hunk 1]
```diff
@@ -521,7 +521,7 @@ export default function Community(props) {
         <section id="community" className="py-12 scrollable">
           <div className="inner">
             <h2 className="text--center">Meet the Community</h2>
-            <p>Pulsar community consists of PMC members and committers. </p>
+            <p>Pulsar community consists of PMC members, committers and contributors. </p>
```
Comment (michaeljmarshall) [inline site2/website-next/src/pages/community.js:524]: Given the roles section of the Apache website, https://www.apache.org/foundation/how-it-works.html#roles, does it make sense to say "Pulsar Development Community" here? My main point is that users are part of the community.

**Review by michaeljmarshall — APPROVED** [review]
LGTM. I left a comment to ask about the "Meet the Community" section.

File: site2/website-next/src/pages/community.js (hunk 1 above)
Comment (lhotari) [reply site2/website-next/src/pages/community.js:524]: That's true. The "Meet the community" page isn't aligned. I'd suggest to create a separate issue to deal with the content on the community page.

**Pull request merged**